Tests/LibWeb/Text/input/DOM/Offset-of-empty-inline-element.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        const element = document.getElementById("empty");
+        const top = element.offsetTop;
+        const left = element.offsetLeft;
+        println("PASS (Didn't crash)");
```

Comment:
let's print `offsetTop` and `offsetLeft` values here? then this test will be even more useful.